ash/src/extensions/khr/video_queue.rs

```diff
@@ -0,0 +1,182 @@
+//! <https://registry.khronos.org/vulkan/specs/1.3-extensions/man/html/VK_KHR_video_queue.html>
+
+use crate::prelude::*;
+use crate::vk;
+use crate::RawPtr as _;
```

Comment:
I like importing trait _functionality_ with `as _`, but we should apply this pattern globally across the repo if we commit to it.

Is there a `clippy` lint that could help us with such a migration?

Reply:
Should I remove this or leave it? Or make a separate PR with the lint, and then rebase once that's merged?

Reply:
Removed the `as _`, based on https://github.com/ash-rs/ash/pull/965#discussion_r1865861437.